src/web/shared/KMZExport.ts

```diff
@@ -142,7 +163,7 @@ export class KMLDocument {
      *
      * @returns {Promise<string>} the KML document as a string
      */
-    async getKML() {
+    async getKML(): Promise<string> {
```

Comment:
Is adding the return type in the function declaration something we should start doing for all functions?